packages/aio-commerce-lib-core/README.md

```diff
@@ -0,0 +1,155 @@
+# @adobe/aio-commerce-lib-core
+
+> [!WARNING]
+> This package is a work in progress and is not yet ready for use yet. You may be able to install it, but if you do, expect breaking changes.
+
+A utility library for handling result objects and error management in Adobe Commerce SDKs.
+
+## Installation
+
+```shell
+npm install @adobe/aio-commerce-lib-core
+```
+
+## Usage
+
+In your App Builder application, you can use the library to bubble up errors and handle results in a consistent way. The reason for this library is to provide a structured way to handle success and failure cases without relying on exceptions, which can lead to cleaner and more maintainable code.
+
+- **Type safety:** Forces you to handle both success and error cases.
+- **No exceptions:** Avoids throwing/catching errors for control flow.
+- **Clarity:** Makes intent explicit and code easier to follow.
+
+### Result
+
+Within every AppBuilder action you can use the `Result<T, E>` type to handle success and failure cases. The `Result` type is a union type that can either be a success with a value of type `T` or a failure with an error of type `E` that extends `ErrorType`.
+
+#### Signatures
+
+- `isOk(result: Result<T, E>): result is { ok: true, value: T }`
+- `isErr(result: Result<T, E>): result is { ok: false, error: E }`
+- `unwrap(result: Result<T, E>): T throws Error`
+- `unwrapErr(result: Result<T, E>): E throws Error`
+- `ok(value: T): Result<T, E>`
+- `err(error: E): Result<T, E>`
+
+#### Use case
+
+- `isOk` and `isErr` are type guards that let you branch logic based on the result.
+- `unwrap` returns the value if the result is a success, otherwise `undefined`.
```

Comment:
otherwise it throws

Reply:
done in 1cafa1648820e0e393afe17df8d94c3e553bfbcc